docs/Controllers.md

```diff
@@ -0,0 +1,87 @@
+# Controllers
+
+_(c) AMWA 2022, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
+
+## Informative
+
+A Controller is Client software that interacts with the NMOS APIs to discover, connect and manage devices within a networked media system.
+
+* This document includes normative references to be followed when implementing a Controller.
+* This document covers how the Controller interacts with the NMOS APIs only.
+It does not cover other features of the Controller software, such as presentation.
+* This document does not cover any requirements relating to where a Controller is additionally acting as a Node (e.g. receiving monitoring information via IS-07).
+
+## General
+
+### User
+Where this document refers to the "user" of a Controller, this includes both human operators who drive the Controller manually and automation systems that drive the Controller programmatically.
+
+### HTTP APIs
+#### Trailing Slashes
+APIs may advertise URLs with or without a trailing slash.
+Controllers appending paths to `href` type attributes MUST support both cases, avoiding doubled or missing slashes.
+
+Controllers performing requests other than GET or HEAD (i.e PUT, POST, DELETE, OPTIONS etc.) MUST use URLs with no trailing slash present.
+
+#### API Version
+Controllers are responsible for identifying the correct API version they require.
+
+Implementers of Controllers are strongly recommended to support multiple versions of the NMOS APIs simultaneously in order to ease the upgrade process in live facilities.
+
+Later API versions might use URNs which have not yet been defined and so Controllers MUST be tolerant to these.
```

Comment:
Noted - this should be broadened to include other values. It's worth pointing out that the General section of this Controllers document is intended to be common for all the Controller requirements documents across all affected ISs. As such this was intended to be boiler plate (that could be potentially factored into its own document). The clauses in the General section were lifted from existing requirements in IS-04, IS-05 and IS-08.  Therefore, adopting of a 'common' General section will necessarily mean repetition of requirements for certain specs.  The alternative is to tailor the General section to the given spec. This might be preferable if a common set of General requirements feels unworkable.

Reply:
Resolving conversation as now outdated